src/components/TransactionsInterpreter/tests/extended-transaction-mock.ts

```diff
@@ -0,0 +1,93 @@
+import { baseTransactionMock } from '../helpers/tests/base-transaction-mock';
+import {
+  OperationType,
+  ResultType,
+  TransactionOperationActionType,
+  TxActionCategoryEnum,
+  TxActionsEnum,
+  UITransactionType,
+  VisibleTransactionOperationType
+} from '../helpers/types';
+
+export const transactionMock: UITransactionType = {
+  ...baseTransactionMock,
+  // type: TransferTypeEnum.SmartContractResult,
+  action: {
+    name: TxActionsEnum.transfer,
+    category: TxActionCategoryEnum.scCall,
+    arguments: {
+      token: 'token',
+      token1: 'token1',
+      token2: 'token2',
+      transfers: 'transfers',
+      null: null,
+      undefined: undefined
+    }
+  },
+  receiverAssets: {
+    name: 'john doe',
+    description: 'description',
+    iconPng: 'icon.png',
+    iconSvg: 'icon.svg',
+    tags: ['@tag1', '@tag2']
+  },
+  operations: [
+    {
+      name: 'send',
+      type: VisibleTransactionOperationType.egld,
+      action: TransactionOperationActionType.transfer,
+      esdtType: 'FungibleESDT',
+      receiver:
+        'erd1qqqqqqqqqqqqqpgq4gdcg0k83u7lpv4s4532w3au9y9h0vm70eqq6m8qk2',
+      sender: 'erd1wh9c0sjr2xn8hzf02lwwcr4jk2s84tat9ud2kaq6zr7xzpvl9l5q8awmex',
+      value: '1',
+      decimals: 4,
+      identifier: 'EGLD',
+      message: 'message 0'
+    },
+    {
+      name: 'send',
+      type: VisibleTransactionOperationType.egld,
+      action: TransactionOperationActionType.transfer,
+      esdtType: 'FungibleESDT',
+      receiver:
+        'erd1qqqqqqqqqqqqqpgq4gdcg0k83u7lpv4s4532w3au9y9h0vm70eqq6m8qk2',
+      sender: 'erd1wh9c0sjr2xn8hzf02lwwcr4jk2s84tat9ud2kaq6zr7xzpvl9l5q8awmex',
+      value: '1',
+      decimals: 4,
+      identifier: 'EGLD',
+      message: 'message 1'
+    }
+  ] as OperationType[],
```

Comment:
normally you would declare a const above with `const operation: OperationType`, then put it into an array, because casting like this is prone to errors